src/components/EditPage.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import styled from 'styled-components'
+import { connect } from 'react-redux';
+import FormInfo from './shared/FormInfo';
+
+const StyledSection = styled.section`
+    .modal {
+        background: rgba(0, 0, 0, 0.6);
+    }
+
+    .modal-main {
+        position:fixed;
+        width: 400px;
+        top:50%;
+        left:50%;
+        transform: translate(-50%,-50%);
+        padding: 25px;
+        margin: 50px auto;
+        border: 3px solid black;
+        background-color: #424242;
+    }
+
+    .display-block {
+        display: block;
+    }
+
+    .display-none {
+        display: none;
+    }
+`;
+
+const EditPage = (props) => {
+    const showHideClassName = props.showEditPage ? "modal display-block" : "modal display-none";
+
+    const state = {
+        namePage: 'Edit movie',
+        nameButton: 'Save',
+        data: {
+            id: 1,
+            title: 'Moana',
+            date: '2018-07-22',
+            url: 'www.moana.com',
+            genre: 'Comedy',
+            overview: 'overview fake',
+            time: 180
+        }
+    };
```

Comment:
I'm expecting the real data to be edited (even without the real submit action)

Reply:
I used mock data, but I will fix it with real data on 6 task